normandy/recipes/utils.py

```diff
@@ -115,3 +116,33 @@ def sign_data(self, content_list):
                 'public_key': res['public_key'],
             })
         return signatures
+
+
+def verify_signature(data, signature, pubkey):
+    if isinstance(data, str):
+        data = data.encode()
+
+    # add data template
+    data = b'Content-Signature:\x00' + data
+
+    signature = base64.b64decode(un_urlsafe(signature))
```

Comment:
i thought this was doing the opposite of `urlsafe_b64decode` but maybe i'm wrong.